.buildkite/x-pack/elastic-agent/.buildkite/pipeline.xpack.elastic-agent-binary-dra.yml

```diff
@@ -0,0 +1,85 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
```

Comment:
What's the reason for using this path that contains `.buildkite` twice (parent and sibling)? As far as I see, there are already pipelines in the folder `.buildkite/x-pack/elastic-agent`:
- https://github.com/elastic/beats/tree/7.17/.buildkite/x-pack/elastic-agent